src/components/ProjectListing.js

```diff
@@ -38,6 +39,7 @@ export const ProjectListing = ({
     onClose={handleHide}
     scroll="paper"
     aria-labelledby="project-listing-title"
+    maxWidth="md"
```

Comment:
This can be changed if you would like the width even higher, see https://material-ui.com/api/dialog/#props